build/Dockerfile

```diff
@@ -6,36 +6,38 @@ RUN dpkg --add-architecture i386 \
 	&& apt-get update -y \
 	&& apt-get install -y --no-install-recommends ca-certificates curl lib32gcc1 lib32stdc++6 libcurl4-gnutls-dev:i386 \
 	&& apt-get clean \
-	&& rm -rf /var/lib/apt/lists/*
-
-# Install gosu.
```

Comment:
I split those into two layers for caching optimization. These should be two image layers, not one. There's no gain in combining those two into one layer, is there?